apps/judicial-system/web/src/components/DefenderInfo/DefenderInfo.tsx

```diff
@@ -155,17 +188,11 @@ const DefenderInfo: FC<Props> = ({ workingCase, setWorkingCase }) => {
                   RequestSharedWithDefender.COURT_DATE
                 }
                 onChange={() => {
-                  setAndSendCaseToServer(
-                    [
-                      {
-                        requestSharedWithDefender:
-                          RequestSharedWithDefender.COURT_DATE,
-                        force: true,
-                      },
-                    ],
-                    workingCase,
-                    setWorkingCase,
-                  )
+                  handleSetAndSendCaseToServer({
+                    requestSharedWithDefender:
+                      RequestSharedWithDefender.COURT_DATE,
+                    force: true,
```

Comment:
Non-related to the changes: What does this `force` property indicate?

Reply:
As discussed this is to make sure we override the value even if the fields being set already have a value. The reason for the flag is to facilitate autofill with default values that we only want to set once.